src/runtime/accessor.cc

```diff
@@ -316,9 +316,11 @@ GenericTensorAccessorW
 
 #define DIMFUNC(DIM)                                                           \
   template class TensorAccessorR<float, DIM>;                                  \
+  template class TensorAccessorR<double, DIM>;                                 \
```

Comment:
Why is this necessary?

Reply:
Standard explicit template instantiation to fix missing symbols during linking. Not sure _why_ the symbols are missing but erred on side of just adding to get working -- worst case scenario binary is sligtly bigger?

Reply:
I guess my concern here is that I think (@jiazhihao correct me if I'm wrong) we don't actually have support for double-precision tensors (everything is `DT_FLOAT`) so I'm concerned that if something is requiring this that the code requiring it may be incorrect

Reply:
Ah gotcha, yeah if it's that unexpected, can do some digging to try to figure out what's causing it

Reply:
> I guess my concern here is that I think (@jiazhihao correct me if I'm wrong) we don't actually have support for double-precision tensors (everything is DT_FLOAT) so I'm concerned that if something is requiring this that the code requiring it may be incorrect

We plan to add support for `DT_DOUBLE`, `DT_HALF`, and other data types. The current implementation supports some type casts for MT5, and our plan is to add a more systematic support for mix-precision training.

Reply:
@lockshaw sounds like the explicit template instantiation is ok then in this case?

Reply:
It's a small enough thing that I don't want to hold up this PR on it anyway :slightly_smiling_face: